card/src/main/java/com/adyen/checkout/card/CardConfiguration.java

```diff
@@ -60,36 +61,25 @@ public CardConfiguration createFromParcel(@NonNull Parcel in) {
     };
 
     /**
-     * @param shopperLocale         The locale that should be used to display strings and layouts. Can differ from device default.
-     * @param environment           The environment to be used to make network calls.
-     * @param shopperReference      The unique identifier of the shopper.
-     * @param holderNameRequired    If the holder name of the card should be shown as a required field.
-     * @param showStorePaymentField If the component should show the option to store the card for later use.
-     * @param supportCardTypes      The list of supported card brands to be shown to the user.
-     * @param hideCvc               Hides the CVC field on the payment flow so that it's not required.
-     * @param hideCvcStoredCard     Hides the CVC field on the stored payment flow so that it's not required.
+     * @param shopperLocale The locale that should be used to display strings and layouts. Can differ from device default.
+     * @param environment   The environment to be used to make network calls.
+     * @param builder       The builder that contains the card configuration
      */
     CardConfiguration(
             @NonNull Locale shopperLocale,
             @NonNull Environment environment,
             @NonNull String clientKey,
-            boolean holderNameRequired,
-            @NonNull String shopperReference,
-            boolean showStorePaymentField,
-            @NonNull List<CardType> supportCardTypes,
-            boolean hideCvc,
-            boolean hideCvcStoredCard,
-            SocialSecurityNumberVisibility socialSecurityNumberVisibility
+            @NonNull Builder builder
```

Comment:
Why are you passing the builder here instead of the fields themselves?

Reply:
After adding the KcpAuthVisibility config there was 11 params in the constructor. We got a lint error and I thought that was an error we shouldn't suppress because it slowly starts become a mess. This looks cleaner to me.

Reply:
If that's the case then the Builder should be the only parameter in the constructor. 
If you want to make this change I think it's cool, but maybe it should be in a separate PR changing all Configurations at once. wdyt?

Reply:
Makes sense, I'll revert it for this PR.